config/roles.yml

```diff
@@ -271,6 +271,15 @@ cross_cluster_search_remote_full_access:
         - 'indices:admin/shards/search_shards'
         - 'indices:data/read/search'
 
+# Allow users to operate query assistant
+ml_query_assistant_access:
```

Comment:
All permissions should be in the alphabetic order. this is a reason why PR now fails.